README.md

```diff
@@ -44,235 +31,8 @@ JFrog Frogbot is a Git bot that scans your Git repositories for security vulnera
-|     ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/notApplicableHigh.png)<br>    High      | $\color{#3CB371}{\textsf{Not Applicable}}$ |lodash:4.17.19 | lodash:4.17.19 | [4.17.21] |
-
-<br>
-
-##### Vulnerability Contextual Analysis
-![](https://raw.githubusercontent.com/jfrog/frogbot/master/images/pr-vuln-contextual-analysis.png)
-
-##### Static Application Security Testing (SAST)
-![](https://raw.githubusercontent.com/jfrog/frogbot/master/images/pr-sast.png)
-
-##### Infrastructure as Code scans (IaC)
-![](https://raw.githubusercontent.com/jfrog/frogbot/master/images/pr-iac.png)
-
-##### Validate Dependency Licenses
-When Frogbot scans newly opened pull requests, it checks the licenses of any new direct project dependencies introduced by the pull request. If Frogbot identifies licenses that are not listed in a predefined set of approved licenses, it appends a comment to the pull request providing this information.
-
-![](https://raw.githubusercontent.com/jfrog/frogbot/master/images/violated-licenses.png)
-
-#### Secrets Detection
-When Frogbot detects secrets that have been inadvertently exposed within the code of a pull request, it promptly triggers an email notification to the user who pushed the corresponding commit. The email address utilized for this notification is sourced from the committer's Git profile configuration. Moreover, Frogbot offers the flexibility to direct the email notification to an extra email address if desired. To activate email notifications, it is necessary to configure your SMTP server details as variables within your Frogbot workflows.
-
-![](https://raw.githubusercontent.com/jfrog/frogbot/master/images/secrets-email.png)
-
-</details>
-
-<details>
-  <summary>Scanning repositories</summary>
-
-### Automatic pull requests creation
-Frogbot scans your Git repositories periodically and automatically creates pull requests for upgrading vulnerable dependencies to a version with a fix.
-
-![](./images/fix-pr.png)
-
-### Adding Security Alerts
-  
-For GitHub repositories, issues that are found during Frogbot's periodic scans are also added to the [Security Alerts](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/managing-code-scanning-alerts-for-your-repository) view in the UI. 
-The following alert types are supported:
-
-#### 1. CVEs on vulnerable dependencies
-
-![](./images/github-code-scanning.png)
-
-![](./images/github-code-scanning-content.png)
-
-#### 2. Secrets that are exposed in the code
-![](./images/github-code-scanning-secrets-content.png)
-
-#### 3. Infrastructure as Code (Iac) issues on Terraform packages
-![](./images/github-code-scanning-iac-content.png)
-
-</details>
-
-</details>
+## 🏁 Getting stated
```

Comment:
```suggestion
## 🏁 Getting started
```

Reply:
Thanks @omerzi!
Creating a follow-up PR to fix this.